pop_tools/grid.py

```diff
@@ -140,6 +146,15 @@ def get_grid(grid_name, scrip=False):
     ), f'unexpected dims in region_mask file: {grid_attrs["region_mask_fname"]}'
     REGION_MASK = region_mask_flat.reshape(grid_attrs['lateral_dims']).astype(np.int32)
 
+    # derive depth of columns of ocean T-points and U-points
+    KMT_reidx = KMT - 1
+    KMT_reidx[KMT_reidx == -1] = 0
```

Comment:
This deals with the issues @klindsay28 mentioned about `IndexErrors`. Since the index from KMT is not zero-based. 

Question: Would one anticipate the `HT`/`HU` output for land to be `np.nan` or 0? Currently it's 0.